data/syntax-highlighting/vim/syntax/meson.vim

```diff
@@ -123,6 +123,13 @@ syn keyword mesonBuiltin
   \ vcs_tag
   \ warning
 
+" Built-in object
+syn keyword mesonBuiltinObj
```

Comment:
It's not completely obvious to me why this is better than, say, adding a script that merges an override list to the refman output and adds them all to a single variable.

Reply:
It just seemed faster that way, but adding the feature to `refman` is definitely better. It give it a shot.